src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -33,14 +26,964 @@ to make sure IE does not go into compatibility mode.
+**[Mimi note: Drop down here]**
 
-One thing is the same across all our JavaScript SDKs and that's how you add or remove _Integrations_, e.g.: for `@sentry/browser`.
+**[Mimi note: Dropdown which affects install instructions are shown (if nothing is provided in the URL, defaults to browser)]**
 
-### Adding an Integration
+- In the browser
+- Angular
+- Electron
+    - Includes some minidump explanation, maybe redirects to the native page
+- Ember
+- Node.js
+- React
+    - Includes a note pointing to React Native
+- Vue
+- Cordova
+- Ionic
+
+The quickest way to get started is to use the CDN hosted version of the JavaScript browser SDK:
+
+```
+<script src="https://browser.sentry-cdn.com/4.5.3/bundle.min.js" crossorigin="anonymous"></script>
+
+```
+
+&nbsp;
+### Configuring the SDK with your Data Source Name
+After you've completed setting up a project in Sentry, Sentry will give you a value which we call a _DSN_ or _Data Source Name_. It looks a lot like a standard URL, but it’s just a representation of the configuration required by the Sentry SDKs. It consists of a few pieces, including the protocol, public key, the server address, and the project identifier.
+
+**[Mimi note: Drop down here]**
+
+You should `init` the Sentry Browser SDK as soon as possible during your page load:
+
+```
+Sentry.init({ dsn: 'https://<key>@sentry.io/<project>' });
+```
+
+Most SDKs will now automatically collect data if available; some require extra configuration as automatic error collecting is not possible due to platform limitations.
+
+&nbsp;
+## Capturing Errors
+### Capturing Errors/ Exceptions
+In JavaScript, you can pass an error object to `captureException()` to get it captured as an event.
+
+```
+try {
+  aFunctionThatMightFail();
+} catch (err) {
+  Sentry.captureException(err);
+}
+```
+
+{% capture __alert_content -%}
+It's possible to throw strings as errors. In this case, the Sentry SDK will not record tracebacks.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="warning"
+%}
+
+&nbsp;
+### Automatically Capturing Errors
+Sentry attaches global handlers to capture uncaught exceptions and unhandled rejections.
+
+**[Mimi note: Code Snippet]**
+
+&nbsp;
+### Automatically Capturing Errors with Promises
+By default, Sentry for JavaScript captures unhandled promise rejections as described in the official ECMAScript 6 standard.
+
+**[Mimi note: Example Code]**
+
+Configuration may be required if you are using a third-party library to implement promises:
+
+Most promise libraries have a global hook for capturing unhandled errors. You may want to disable default behavior by setting `captureUnhandledRejections` option to `false` and manually hook into such event handler and call `Raven.captureException` or `Raven.captureMessage` directly.
```

Comment:
I think this came from my sloppy cutting & pasting. "Sentry" should replace "Raven" everywhere, right?